packages/roosterjs-editor-api/lib/format/removeLink.ts

```diff
@@ -3,7 +3,11 @@ import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
 import { normalizeEditorPoint, unwrap } from 'roosterjs-editor-dom';
 import { Editor } from 'roosterjs-editor-core';
 
-export default function removeLink(editor: Editor): void {
+/**
+ * Remove link at selection, if no links at selection, do nothing
```

Comment:
explain what will happen when selection contains multiple links.
explain what will happen if only part of a link is selected. e.g. <a>THIS IS {Sel Start} A LINK </a> {Sel End}

Reply:
Added.